packages/manager/modules/sms/src/sms/components/service-status/service-status.html

```diff
@@ -0,0 +1,66 @@
+<div class="col-sm-4">
+    <oui-tile
+        data-heading="{{ 'sms_component_serviceStatus_title' | translate }}"
+        data-loading="$ctrl.loading"
+    >
+        <oui-tile-definition
+            data-term="{{ 'sms_component_serviceStatus_label_status' | translate }}"
+        >
+            <oui-tile-description>
+                <span
+                    class="oui-badge"
+                    data-ng-class="{
+                    'oui-badge_error': $ctrl.status === ('ERROR' || 'disable'),
+                    'oui-badge_success': $ctrl.status === ('OK' || 'enable'),
+                    'oui-oui-badge_info': $ctrl.status === ('INSTALLING' || 'UPDATING_IPS'),
+                }"
+                    data-translate="{{ 'sms_component_serviceStatus_status_' + $ctrl.status }}"
+                >
+                </span>
+            </oui-tile-description>
+        </oui-tile-definition>
+        <oui-tile-definition data-ng-if="$ctrl.isSmppAccount">
+            <oui-tile-description>
+                <oui-button
+                    data-variant="secondary"
+                    data-disabled="$ctrl.status !== 'OK'"
+                    data-on-click="$ctrl.openResetPasswordModal()"
+                >
+                    <span
+                        data-translate="sms_component_serviceStatus_button_reset_password"
+                    >
+                    </span>
+                </oui-button>
+            </oui-tile-description>
+        </oui-tile-definition>
+    </oui-tile>
+</div>
+<div data-ng-if="$ctrl.isOpenModal">
+    <div class="modal d-block">
+        <div class="modal-dialog modal-dialog-centered modal-dialog-scrollable">
+            <div class="modal-content">
+                <form novalidate name="resetSmppPasswordForm">
```

Comment:
**nitpick:** Since there are no fields in th modal, we can remove the `form`.

Reply:
It's fixed, i removed it.